minio/api.py

```diff
@@ -1400,8 +1404,9 @@ def _presigned_get_partial_object(self, bucket_name, object_name,
                                  expires=int(expires.total_seconds()))
         return presign_url
 
-    def _do_put_multipart_object(self, bucket_name, object_name, part_metadata,
-                                 upload_id='', part_number=0, metadata=None):
+    def _do_put_object(self, bucket_name, object_name, part_data,
+                       part_size, upload_id='', part_number=0,
+                       metadata=None):
         """
         Initiate a multipart PUT operation for a part number.
```

Comment:
Can you update this block? as I understood this function can upload a single put or a part